src/internal/kopia/cleanup_backups.go

```diff
@@ -144,6 +179,11 @@ func cleanupOrphanedData(
 		}
 	}
 
+	logger.Ctx(ctx).Debugw(
```

Comment:
nit: Could you please change this to `Infow`? It could be an important breadcrumb to look back at, in case we ever hit a regression and accidentally delete some data.